include/exadg/convection_diffusion/user_interface/parameters.h

```diff
@@ -318,6 +318,18 @@ class Parameters
 
   // use 3/2 overintegration rule for convective term
   bool use_overintegration;
+
+  /**************************************************************************************/
+  /*                                                                                    */
+  /*                            ELEMENTWISE INVERSE MASS                                */
+  /*                                                                                    */
+  /**************************************************************************************/
```

Comment:
I think we do not need a separate section for elementwise inverse mass. I think we could add these parameters to the SOLVER parameters above.

Another question is why you introduce these parameters here given that simplex meshes have not been realized so far for the ConvDiff module? Of course, one could use the elementwise iterative solver also for hex-elements, but there we know that the explicit matrix-free inverse mass operator is most efficient.

Generally, my perspective is to only introduce parameters in the user interface if they are relevant. So I would suggest to introduce these parameters once simplex is realized for ConvDiff. Since these parameters will then already exist in the IncNS module, it will be easy to copy that code to ConvDiff (once/if simplex will be realized for ConvDiff).